### PR TITLE
c-struct typings

### PR DESCRIPTION
**Pull request by benallfree**
Greetings, this PR introduces typings for [c-struct](https://www.npmjs.com/package/c-struct)

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## c-struct ([<kbd>unpkg</kbd>](https://unpkg.com/browse/c-struct@latest/))
was missing the following properties:
1. packers
1. unpackers
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9268c81d90d386fb01b6f639135748e7ca456fb7
</p>

File: types/c-struct/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+// Type definitions for c-struct 0.0
+// Project: https://github.com/xdenser/node-struct#readme
```
Comment (amcasey) [inline types/c-struct/index.d.ts:2]: https://github.com/majimboo/c-struct#readme?

File: types/c-struct/tslint.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-class": false
```
Comment (amcasey) [inline types/c-struct/tslint.json:4]: If a lint rule absolutely must be suppressed, please use a line-level suppression to make it easier to review.

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/c-struct/index.d.ts (hunk 1 above)
Comment (benallfree) [reply types/c-struct/index.d.ts]: Good catch :)

**Review by amcasey — APPROVED** [review]

**Comment by benallfree** [comment]
ready to merge

**Pull request merged**